brpc-java-examples/brpc-java-core-examples/src/main/proto/echo3.proto

```diff
@@ -0,0 +1,18 @@
+syntax="proto3";
+
```

Comment:
if you use proto3 syntax, the proto files of built-in protocols on brpc-java-core can also modify to proto3.